gemfiles/rails_70/Gemfile.lock

```diff
@@ -371,6 +377,7 @@ GEM
 PLATFORMS
   arm64-darwin-21
   arm64-darwin-22
+  ruby
```

Comment:
this is being added by my local configuration. I have a global `force_ruby_platform` setting